api/src/main/java/io/opencensus/metrics/export/Distribution.java

```diff
@@ -163,6 +132,104 @@ public static Distribution create(
    */
   public abstract List<Bucket> getBuckets();
 
+  /**
+   * The bucket options used to create a histogram for the distribution.
+   *
+   * @since 0.17
+   */
+  @Immutable
+  public abstract static class BucketOptions {
+
+    private BucketOptions() {}
+
+    /**
+     * Returns a ExplicitOptions {@link ExplicitOptions}.
```

Comment:
nit: remove the first "ExplicitOptions".

Reply:
Done